conduit/metrics.py

```diff
@@ -83,27 +86,44 @@ def __call__(self, y_pred: Tensor, *, y_true: Tensor) -> R:
 
 
 def _pdist_1d(x: Tensor) -> Tensor:
-    return torch.pdist(x.view(-1, 1)).squeeze()
+    res = torch.pdist(x.view(-1, 1)).squeeze()
+    return res
```

Comment:
??

Reply:
Ah I split up the return for the purpose of testing; the squeeze was causing an assert to fail